amr-wind/equation_systems/sdr/SDR.H

```diff
@@ -44,6 +45,46 @@ struct SDR : ScalarTransport
     static constexpr bool need_nph_state = true;
 };
 
+template<>
+struct TurbulenceOp<SDR>
+{
+    TurbulenceOp(turbulence::TurbulenceModel& tmodel,
+                 PDEFields& fields)
+        : m_tmodel(tmodel), m_fields(fields)
+    {}
+
+    void operator()()
+    {
+        auto& mueff = m_fields.mueff;
+        m_tmodel.update_scalar_diff(mueff, "sdr");
+    }
+
+    turbulence::TurbulenceModel& m_tmodel;
+    PDEFields& m_fields;
+};
+
+/** Implementation of post-processing actions for SDR equation
+ *  \ingroup sdr_eqn
+ */
+template <>
+struct PostSolveOp<SDR>
```

Comment:
Move to `sdr_ops`?

Reply:
https://github.com/Exawind/amr-wind/pull/322/commits/cac0cf80a0d3db19f2afc6164c91424bf91902b7